metrics/src/main/java/io/opencensus/metrics/Value.java

```diff
@@ -229,14 +239,6 @@ public static DistributionValue create(
     /**
      * Returns the aggregated sum of squared deviations.
      *
-     * <p>The sum of squared deviations from the mean of the values in the population. For values
-     * x_i this is:
-     *
-     * <p>Sum[i=1..n]((x_i - mean)^2)
```

Comment:
It is fine to define the "sum of squared deviations", I just thought that we shouldn't specify that it is calculated in one pass.